crypto/README.md

```diff
@@ -0,0 +1,13 @@
+# Crypto
+
+A collection of functions used to interpret and extract various components of the circuits. 
+
+# Features
+The main features of this crate at the moment are:
+- Ability to serialize/deserialize a SNARK wrapped FRI proof (this is the type of proof used on L1 for boojum)
+- Calculate the hash of a given verification key
+
+While the deserializer/verification key hashing is new, the [serialization code](src/serialize.rs) comes from the [solidity plonk verifier repo](https://github.com/matter-labs/solidity_plonk_verifier/blob/82f96b7156551087f1c9bfe4f0ea68845b6debfc/codegen/src/lib.rs#L81). It is restated here to avoid mismatches in type versions.
+
```

Comment:
please also mention about the  scheduler_key.json - where is it taken from, and how to refresh if needed.